src/pages/components/timeline/TimelinePoint.tsx

```diff
@@ -1,4 +1,4 @@
-import { TEXTS } from 'src/resources/texts'
+import { t } from 'i18next'
```

Comment:
```suggestion
```

Reply:
I did not remove the i18next modal, because I'm still using it in pointTypeToDescription.